src/app/modules/scheduler/components/scheduler-modal/scheduler-preview-column/scheduler-preview-column.component.html

```diff
@@ -1,9 +1,16 @@
 <section class="preview-header">
   <h4 class="crontab-header">
     {{ 'Schedule Preview' | translate }}
-    <button class="close-btn" mat-icon-button ixTest="close-scheduler" [matDialogClose]>
-      <ix-icon name="cancel"></ix-icon>
-    </button>
+    <div class="close-wrapper" [attr.aria-label]="'Close scheduler' | translate">
```

Comment:
Other buttons has aria-label without a wrapper. Why is it needed here?

Reply:
Don't know why but it doesn't work on the button or icon itself. Only works on them if not used as an input property. But we need to use it as an input property to provide translation logic. So it works there if I use it like `aria-label="Close scheduler"` but not as `[attr.aria-label]="'Close scheduler' | translate"`